tests/unit/i18n-t-test.js

```diff
@@ -54,3 +54,12 @@ test("applies custom pluralization rules", function(assert) {
   assert.equal(i18n.t('pluralized.translation', { count: 1 }), 'One Click');
   assert.equal(i18n.t('pluralized.translation', { count: 2 }), '2 Clicks');
 });
+
+test("applies provided default translation in cascade when main one is not found", function(assert) {
+  const i18n = this.subject({ locale: 'en' });
+  const defaultsChain = ['with.pretty-good-interpolations', 'with.interpolations'];
+
+  assert.equal(i18n.t('with.great-interpolations', { clicks: 8, default: defaultsChain }), 'Clicks: 8', 'default can be an array');
+  assert.equal(i18n.t('with.great-interpolations', { clicks: 8, default: 'with.interpolations' }), 'Clicks: 8', 'default can be an string');
+  assert.equal(i18n.t('not.yet.translated', { clicks: 8, default: ['not.translated.either'] }), 'Missing translation: not.yet.translated');
+});
```

Comment:
I'd also like to see a test that the missing event happens with the *primary* key.